plutus-metatheory/src/Algorithmic/Erasure/Reduction.lagda

```diff
@@ -1,284 +0,0 @@
-\begin{code}
-{-# OPTIONS --rewriting #-}
-
-module Algorithmic.Erasure.Reduction where
```

Comment:
We used to have an implementation of untyped reduction and a theorem connecting typed reduction and untyped reduction which was in this file. Currently we have an implementation of untyped CEK instead and the connection between typed and untyped CEK is via testing. 

It may be that we reintroduce this later but I don't think this is a strong enough reason to keep this around.